tensorflow/python/ops/nn_impl.py

```diff
@@ -432,7 +432,8 @@ def relu_layer(x, weights, biases, name=None):
 @dispatch.add_dispatch_support
 def swish(features, beta=1.0):
   # pylint: disable=g-doc-args
-  """Computes the SiLU or Swish activation function: `x * sigmoid(beta * x)`.
+  """The Sigmoid Linear Unit (SiLU), also known as the Swish activation function,
+  is computed as follows `x * sigmoid(beta * x)`.
```

Comment:
Docstrings are one short line and then more description in new paragraphs. Please reword.